doc/orchestrator.rst

```diff
@@ -165,3 +165,399 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+requirements. Below is a general example demonstrating the process of launching
+multiple databases with unique identifiers and distributing data between them.
+
+.. note::
+
+  To run the following example, configure your environment by following
+  :ref:`Site Installation <site_installation>` then build SmartSim
+  by following the :ref:`Build Steps <from-pypi>`.
+
+Workflow Initialization and Multi Orchestrator Setup
+----------------------------------------------------
+
+Defining workflow stages requires the utilization of functions associated
+with the ``Experiment`` object. The Experiment object is intended to be instantiated
+once and utilized throughout the workflow runtime. Begin by creating a ``Experiment``
+object and assign it to the ``exp`` variable. In this example, we instantiate the
+``Experiment`` object with the name ``getting-started-multidb``. We setup the SmartSim
+``logger`` as well for use within the Experiment. For more information on the SmartSim,
+click here.
+
+.. code-block:: python
+
+  from smartsim import Experiment
+  from smartredis import Client
+  from smartredis.log import get_logger
+  import numpy as np
+
+  exp = Experiment("getting-started-multidb", launcher="auto")
+  logger = get_logger("Multidb Experiment Log")
+
+In the context of this ``Experiment``, it's essential to create and launch
+the databases as a preliminary step before any other components. We aim
+to showcase the multi-database automation capabilities of SmartSim, so we
+create two types of Orchestrators in the workflow: a co-located Orchestrator and a
+clustered Orchestrator.
+
+To create an Orchestrator, utilize the ``create_database()`` function located
+in the ``Experiment`` class. When launching more than one Orchestrator, the
+function requires specifying a unique database identifier
+argument named ``db_identifier``. The ``db_identifier``
+serves as a unique identifier for each database being launched. Usage of the ``db_identifier``
+is show in the example below.
+
+Here's how to create the databases:
+
+For the co-located database:
+
+.. code-block:: python
+
+  colocated_orch = exp.create_database(port=6379, db_nodes=1, interface="ib0", db_identifier="single_shard_db_identifier")
+  exp.generate(colocated_orch, overwrite=True)
+
+For the clustered database:
+
+.. code-block:: python
+
+  clustered_orch = exp.create_database(port=6380, db_nodes=3, interface="ib0", db_identifier="clustered_db_identifier")
+  exp.generate(clustered_orch, overwrite=True)
+
+.. note::
+  The ``exp.generate()`` function accepts entities within the workflow, such as Model,
+  Ensemble, and Orchestrator. This function organizes the outputs of these entities
+  under the associated Experiment folder. In this example, the Experiment folder is
+  named ``getting-started-multidb/``. Within this folder, two Orchestrator subfolders will
+  be created, namely ``single_shard_db_identifier/`` and ``clustered_db_identifier/``. It's
```

Comment:
Combine these two sentences.

"Calling ``exp.generate`` will create two subfolders (one for each Orchestrator created in the previous step) whose names are based on the ``db_identifier`` of that Orchestrator